packages/smooth_app/lib/cards/data_cards/image_upload_card.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:io';
+import 'package:flutter/material.dart';
+
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/model/Product.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:image_cropper/image_cropper.dart';
+
+class ImageUploadCard extends StatefulWidget {
+  const ImageUploadCard({
+    Key key,
+    this.product,
+    this.imageField,
+    this.buttonText,
+  }) : super(key: key);
+
+  final Product product;
+  final ImageField imageField;
+  final String buttonText;
+
+  @override
+  _ImageUploadCardState createState() => _ImageUploadCardState();
+}
+
+class _ImageUploadCardState extends State<ImageUploadCard> {
+  File _image;
+  final ImagePicker picker = ImagePicker();
+
+  Future<void> _getImage() async {
+    print('getImage');
```

Comment:
Please remove the `print` statements.

Reply:
ok. how should we log helpful messages about what the app does in the debug console?

Reply:
That's a good question.
I would say in a first approach that `print` statements are when you are debugging your code - I use tons of them, to be removed when you're done with the tests and when you create a PR.
Another way to deal with that would be to use something like [`logger`](https://pub.dev/packages/logger). I guess that should be part of the general development rules.